credentials/apps/edx_credentials_extensions/edly_credentials_app/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
```

Comment:
why this file if there are no views?

Reply:
similarly, here kept it there if in future we need to use this.